api/config/service/aws/v1/aws.proto

```diff
@@ -9,9 +9,26 @@ import "validate/validate.proto";
 message Config {
   repeated string regions = 1 [ (validate.rules).repeated = {min_items : 1} ];
   ClientConfig client_config = 2;
+  ClientSettings client_settings = 3;
 }
 
 message ClientConfig {
   // If not set explicity, retries default to 0
   int32 retries = 1 [ (validate.rules).int32.gte = 0 ];
 }
+
+message ClientSettings {
+  DynamodbScalingLimits dynamodb_scaling_limits = 1;
+}
+
+message DynamodbScalingLimits {
+  // defaults to AWS default quotas if not set
+  // https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Limits.html#default-limits-throughput-capacity-modes
+  int64 max_read_capacity_units = 1 [ (validate.rules).int64.gte = 1 ];
+  int64 max_write_capacity_units = 2 [ (validate.rules).int64.gte = 1 ];
```

Comment:
Doesn't the validation mark this as a required field in the config?

Reply:
Good catch, I'll add `ignore_empty` constraint to the validation